src/components/list/list.scss

```diff
@@ -138,11 +138,13 @@ md-list-item {
       align-items: center;
       justify-content: flex-start;
 
-      padding: 0;
+      padding: 0 16px;
       margin: 0;
 
+      // The button should not inherit the parents background color.
+      background-color: initial;
```

Comment:
I do not understand the **need** for this setting. I see no difference with or without this setting.